pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "delatore"
-version = "0.2.3"
+version = "0.3.0"
```

Comment:
It's not that big change :(

Reply:
This pr includes previous version. That's why decided to change on 0.3.0